Ginger/Ginger/AutomatePageLib/AddActionMenu/WindowExplorer/Common/ControlActionsPage_New.xaml.cs

```diff
@@ -212,6 +212,18 @@ void SetPlatformBasedUIUpdates()
             else
             {
                 DefaultAction.Context = mContext;
+
```

Comment:
we shouldn't have specific platforms / elementTypes if-else and switches in generic classs/controls!!